themes/default/content/docs/get-started/install/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Download and Install"
-meta_desc: This page contains detailed instructions for installing Pulumi.
+title: "Download Pulumi"
```

Comment:
slight hesitation about loosing the install language here

Reply:
I can revert, always unfortunate when we make one thing sound like two, IMHO they're basically the same action from the user's perspective. But I'm only like 10% on this and happy to keep as is if you feel the old phrasing is better.